src/omero_cli_render.py

```diff
@@ -668,7 +668,10 @@ def set(self, args):
                 "Updated rendering settings for Image:%s" % img.id)
             if not args.skipthumbs:
                 self._generate_thumbs([img])
-            img._closeRE()
+            try:
+                img._closeRE()
+            except Exception as e:
```

Comment:
@will-moore does  `_closeRE()` handle exceptions internally?

In addition, I think the question of https://github.com/ome/omero-cli-render/pull/29#discussion_r260632056 was to handle any exception thrown in the previous calls e.g. in `img.saveDefaults()` to ensure the service would be closed

```
    try:
        ...
        img.saveDefaults()
        ...
    finally:
        img._closeRE()
```

Reply:
How do you actually know that saveDefaults() can throw an exception? I'll still find it unbelievabe difficult to do use the cli and/or BlitzGatway methods. E.g. https://downloads.openmicroscopy.org/omero/5.4.10/api/python/omero/omero.gateway.html#omero.gateway._ImageWrapper.saveDefaults doesn't mention any exception, it doesn't even say anything about what the method actually does.